api/_globals/clearInterval.json

```diff
@@ -92,10 +92,10 @@
               "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": "≤4"
+              "version_added": "4"
             },
             "safari_ios": {
-              "version_added": "≤3"
+              "version_added": "5"
```

Comment:
What is this based on? Depending on when the change was made in WebKit, I think this should be 2, 3 or 3.2.

Reply:
This is in relation to when worker support was added, which is Safari iOS 5 by our data!

Reply:
Oh right, I keep forgetting :)